<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>627</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Microsoft JhengHei UI</family>
    <pointsize>11</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="0" rowspan="2" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="spacing">
      <number>8</number>
     </property>
     <item>
      <widget class="QPushButton" name="new_shot">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
border-radius: 12px;
background-color:rgb(96, 68, 100);
}
QPushButton:hover{
background-color:rgb(127, 104, 130);
}
</string>
       </property>
       <property name="text">
        <string>新建镜头</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="enter_shot">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
border-radius: 12px;
background-color:rgb(96, 68, 100);
}
QPushButton:hover{
background-color:rgb(127, 104, 130);
}
</string>
       </property>
       <property name="text">
        <string>进入镜头</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="current_shot">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
border-radius: 12px;
background-color:rgb(96, 68, 100);
}
QPushButton:hover{
background-color:rgb(127, 104, 130);
}
</string>
       </property>
       <property name="text">
        <string>当前镜头</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="set_proj_hip">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
border-radius: 12px;
background-color:rgb(96, 68, 100);
}
QPushButton:hover{
background-color:rgb(127, 104, 130);
}
</string>
       </property>
       <property name="text">
        <string>设JOB为HIP</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QLabel" name="project_path">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Project Path</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>80</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>19</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(186, 181, 200);
border-style: none;
border-radius: 8px;
border-width: 2px;
border-color: rgb(159, 155, 204);
background-color: qlineargradient(spread:pad, x1:0.062, y1:0.0962273, x2:0.994, y2:0.937, stop:0 rgba(75, 73, 140, 255), stop:1 rgba(110, 87, 135, 255))</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;WES&lt;/span&gt; PROJECT BROWSER&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QListWidget" name="hiplist">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 12px;
border-width:0px;
background-color: rgb(13, 13, 13);</string>
     </property>
     <item>
      <property name="text">
       <string>xxx.hip</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0,0">
     <property name="spacing">
      <number>3</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="scene_label">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
        </font>
       </property>
       <property name="text">
        <string>场次</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="scene">
       <property name="minimumSize">
        <size>
         <width>15</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox {
font: 11pt &quot;Microsoft YaHei UI&quot;;
}</string>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="shot_label">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>55</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
        </font>
       </property>
       <property name="text">
        <string>镜头号</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="shot">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox {
font: 11pt &quot;Microsoft YaHei UI&quot;;
}</string>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="content_label">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
        </font>
       </property>
       <property name="text">
        <string>内容</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="content">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>30</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 5px;</string>
       </property>
       <property name="text">
        <string>vfx</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0" colspan="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="0" colspan="2">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="10" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>7</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <widget class="QPushButton" name="save_small_ver">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>24</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
border-radius: 12px;
background-color:rgb(75, 74, 115);
}
QPushButton:hover{
background-color:rgb(109, 109, 150);
}
</string>
       </property>
       <property name="text">
        <string>保存小版本</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save_big_ver">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>24</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
border-radius: 12px;
background-color:rgb(75, 74, 115);
}
QPushButton:hover{
background-color:rgb(109, 109, 150);
}
</string>
       </property>
       <property name="text">
        <string>保存大版本</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="load_hip">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>10000</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
border-radius: 12px;
background-color:rgb(75, 74, 115);
}
QPushButton:hover{
background-color:rgb(109, 109, 150);
}
</string>
       </property>
       <property name="text">
        <string>打开工程</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="open_folder">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>10000</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
border-radius: 12px;
background-color:rgb(75, 74, 115);
}
QPushButton:hover{
background-color:rgb(109, 109, 150);
}
</string>
       </property>
       <property name="text">
        <string>打开文件夹</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255,0);</string>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QComboBox" name="browse_mode">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox {
border-radius: 6px;
}

QComboBox::drop-down {
	
	color: rgb(0, 255, 0);
}
</string>
       </property>
       <item>
        <property name="text">
         <string>Project</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Test</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Custom</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="proj_name">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>23</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 6px;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="new_proj">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>115</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
border-radius: 6px;
background-color:rgb(49, 108, 74);
}
QPushButton:hover{
background-color:rgb(69, 131, 95);
}
</string>
       </property>
       <property name="text">
        <string>新建项目</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>browse_mode</tabstop>
  <tabstop>proj_name</tabstop>
  <tabstop>new_proj</tabstop>
  <tabstop>scene</tabstop>
  <tabstop>shot</tabstop>
  <tabstop>content</tabstop>
  <tabstop>new_shot</tabstop>
  <tabstop>enter_shot</tabstop>
  <tabstop>current_shot</tabstop>
  <tabstop>set_proj_hip</tabstop>
  <tabstop>save_small_ver</tabstop>
  <tabstop>save_big_ver</tabstop>
  <tabstop>load_hip</tabstop>
  <tabstop>open_folder</tabstop>
  <tabstop>hiplist</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
